### PR TITLE
Qutebrowser testing

### PR DESCRIPTION
**Pull request by jason1987d**
Three packages listed in qutebrowser template as needed for testing it that are not currently packaged in void linux.
https://github.com/void-linux/void-packages/pull/42344

#### Testing the changes
- I tested the changes in this PR: briefly
#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): YES
#### Local build testing
- I built this PR locally for my native architecture, (ARCH-LIBC)

### CODE REVIEW

**Comment by paper42** [comment]
Please also enable tests in qutebrowser in this PR so we can see this works

File: srcpkgs/python3-pytest-bdd/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-pytest-bdd'
+pkgname=python3-pytest-bdd
+version=6.1.1
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3 python3-pytest"
```
Comment (mhmdanas) [inline srcpkgs/python3-pytest-bdd/template:7]: python3-pytest already depends on python3:
```suggestion
depends="python3-pytest"
```

File: srcpkgs/python3-pytest-rerunfailures/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-pytest-rerunfailures'
+pkgname=python3-pytest-rerunfailures
+version=11.1.2
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3 python3-pytest"
```
Comment (mhmdanas) [inline srcpkgs/python3-pytest-rerunfailures/template:7]:
```suggestion
depends="python3-pytest"
```

File: srcpkgs/python3-pytest-instafail/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-pytest-instafail'
+pkgname=python3-pytest-instafail
+version=0.4.2
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3 python3-pytest"
```
Comment (mhmdanas) [inline srcpkgs/python3-pytest-instafail/template:7]:
```suggestion
depends="python3-pytest"
```

**Comment by jason1987d** [comment]
I got hold of The-Compiler himself,

> ah, I see the issue... we don't package scripts/dev/ in releases (because 
                      those are development scripts), but we have tests for it
as a quick workaround, can you try just removing 
                      tests/unit/scripts/test_check_coverage.py 
                      tests/unit/scripts/test_problemmatchers.py and 
                      tests/unit/scripts/test_run_vulture.py maybe?

**Comment by jason1987d** [comment]
Word is that qt6 branch will be merged soon.

**Pull request closed without merging**